src/coins.h

```diff
@@ -390,14 +414,18 @@ class CCoinsViewBacked : public CCoinsView
     bool GetNullifier(const uint256 &nullifier) const;
     bool GetCoins(const uint256 &txid, CCoins &coins) const;
     bool HaveCoins(const uint256 &txid) const;
+    bool HaveScInfo(const uint256& scId) const;
```

Comment:
Consider adding c++11 style override specifier to virtual methods

Reply:
Agreed and coded